### PR TITLE
Looks/update front page

### PR DESCRIPTION
**Pull request by joe-gibbs**
This branch makes changes to the front page of the application to update it to cover the work that we've done and the new team working on it.

### CODE REVIEW

File: backend/views/home.pug [hunk 1]
```diff
@@ -77,3 +86,19 @@ block content
 					h6 Ming Sheng Quah
```
Comment (richiealy) [inline backend/views/home.pug:86]: Is Ming Shen Quah meant to be there?

File: backend/views/home.pug [hunk 2]
```diff
@@ -38,6 +38,15 @@ block content
 				.col.s12.m4.l4.center-align
 					img.responsive-img(src="stylesheets/img/network.png")
 			.divider
+			.row.valign-wrapper
+				.col.s12.m8.l8
+					h5 Finding Crimes from Images 
+					p In addition to getting crimes from databases, we also want to dynamically find crimes from Twitter and other sites. 
+					p This means that instead of having to feed data in to the application manually we can simply take the images automatically.
+					p After we've gotten the images, we feed them into Amazon Rekognition to find tags, and then use a co-training model from the images, captions and Rekognition data to determine whether crimes are occuring in the image.
```
Comment (richiealy) [inline backend/views/home.pug]: I'd say `after we've obtained the images, we use them as data for Amazon Rekognition to find tags. Then we use a trained hybrid model to make use of this data along with manually annotated captions to determine whether or not crimes are appearing in images, along with crime severity.`

**Review by richiealy — COMMENTED** [review]
Pretty good. Here are some changes I'd recommend along with a minor catch.

File: backend/views/home.pug (hunk 1 above)
Comment (joe-gibbs) [reply backend/views/home.pug:86]: Pretty sure? That's how it was on the original

File: backend/views/home.pug (hunk 2 above)
Comment (joe-gibbs) [reply backend/views/home.pug]: Done.

File: backend/views/home.pug (hunk 1 above)
Comment (richiealy) [reply backend/views/home.pug:86]: Sorry meant all the above team members as well, wasn't sure if we were keeping them.

Comment (richiealy) [reply backend/views/home.pug:86]: Will leave it for now, I'll go ahead and merge

**Review by richiealy — APPROVED** [review]

**Pull request merged**